docs/targets/azuresentineltarget.md

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
I think this file should be a bit more structured, like https://github.com/triggermesh/triggermesh/blob/main/docs/sources/kafka.md or https://github.com/triggermesh/triggermesh/blob/main/docs/targets/googlestorage.md

Reply:
These are just dev docs.. They are not intended to be the full documentation. Every target i write I include here only the bits to interact with it (if needed) and how to deploy locally. We can decide on a formal structure for these. But idk if this PR is the time.